autoload/leaderf/python/leaderf/manager.py

```diff
@@ -724,6 +724,7 @@ def _createPopupModePreview(self, title, source, line_nr, jump_cmd):
                 options["minheight"] = maxheight
 
             if isinstance(source, int):
+                lfCmd("call bufload(%d)" % source)
```

Comment:
> if the buffer is not loaded the preview will be empty in vim

source is int only when buffer is loaded, for example:
https://github.com/Yggdroot/LeaderF/blob/43433aa8dad1601acf923d3c25fa3325799508aa/autoload/leaderf/python/leaderf/rgExpl.py#L926-L930
did I miss something?

Reply:
From vim doc:
```
bufadd({name})						*bufadd()*
...
		The buffer will not have 'buflisted' set and not be loaded
		yet.  To add some text to the buffer use this: >
			let bufnr = bufadd('someName')
...
```
buffer is not loaded after calling bufadd() .

[LeaderF-git](https://github.com/linjiX/LeaderF-git) will not work if we haven't call bufload().
Or we should call bufload() in Extension side?